codegen/src/main/java/software/amazon/awssdk/codegen/poet/client/specs/JsonProtocolSpec.java

```diff
@@ -303,6 +306,18 @@ public CodeBlock asyncExecutionHandler(IntermediateModel intermediateModel, Oper
         return builder.build();
     }
 
+    private void addMarshaller(CodeBlock.Builder builder, OperationModel opModel, IntermediateModel intermediateModel) {
```

Comment:
If we model the request with Json as I mentioned in the customization.config we will not need this